docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  dicedb:
+    image: dicedb/dicedb:latest
+    ports:
+      - "7379"
+    healthcheck:
+     test: ["CMD", "PING"]
+     interval: 10s
+     timeout: 3s
+     retries: 3
+    networks:
+     - dice-network
+
+  backend:
+    build:
+      context: .
+    ports:
+      - "8080:8080"
+    depends_on:
+      - dicedb
+    environment:
+      - DICE_ADDR=dicedb:7379
```

Comment:
Let's use `DICEDB` everywhere.

Reply:
pardon?

Reply:
he means the naming convention is dicedb and not dice.
like below:
environment:
      - DICEDB_ADDR=dicedb:7379